packages/@react-native-windows/cli/src/config/configUtils.ts

```diff
@@ -164,17 +178,19 @@ export function findDependencyProjectFiles(winFolder: string): string[] {
 function isRnwAppProject(filePath: string): boolean {
   const projectContents = readProjectFile(filePath);
 
+  const rnProjectType = getReactNativeProjectType(
+    tryFindPropertyValue(projectContents, 'ReactNativeProjectType'),
+  );
+
+  if (rnProjectType !== 'unknown') {
+    return true;
+  }
+
   const projectLang = getProjectLanguage(filePath);
   if (projectLang === 'cs') {
-    return (
-      importProjectExists(
-        projectContents,
-        'Microsoft.ReactNative.Uwp.CSharpApp.targets',
-      ) ||
-      importProjectExists(
-        projectContents,
-        'Microsoft.ReactNative.WinAppSDK.CSharpApp.props',
-      )
+    return importProjectExists(
+      projectContents,
+      'Microsoft.ReactNative.Uwp.CSharpApp.targets',
```

Comment:
looks like you removed the check for importing Microsoft.ReactNative.WinAppSDK.CSharpApp.props, was that intended?

Reply:
Yeah I ended up reverting to the logic we had previously because the check for importing WASDK started failing and so the CLI wouldn't build. Using this for now as it works, and plan to update/try checking for a targets file like UWP does once we add a targets file for WASDK.